### DISCUSSION TITLE
Temporary `.scss`-files with the book format

### DISCUSSION THREAD

**Discussion by snhansen** [quarto-dev/quarto-cli#10901] [Q&A] [answered]
### Description

I just updated to the most recent Quarto version (1.6.14) and noticed that my book projects now has files named `temp-1.scss`, `temp-2.scss` in the project folder after rendering (alongside the `.qmd` files). I don't think this is in intentional but I figured I'd better ask here. A minimal example below:

`_quarto.yml`:

	project:
	  type: book

	book:
	  chapters:
		- index.qmd
	  
	format: html

**Comment by cderv** [comment]
Thanks for the report ! Definitely not intentional !

**Reply by cderv** [reply, marked as answer]
I opened 
- https://github.com/quarto-dev/quarto-cli/pull/10905

**Comment by snhansen** [comment]
Resolved in the PR above.
